packages/embark/src/lib/core/events.js

```diff
@@ -1,4 +1,5 @@
 var EventEmitter = require('events');
+const cloneDeep = require('lodash.clonedeep');
```

Comment:
Do we still have lodash as a dependency? We have an individual package from deep cloning.

Reply:
It appears that we have both `lodash.clonedeep` and `clone-deep` in our dependencies 🤷‍♂️